src/Applications/GEOSdas_App/GEOSdas.csm

```diff
@@ -6082,9 +6082,9 @@ endif
 #------------------------------------ 
 
   if ( $LDAS_ANA ) then
-    mkdir -p               $FVHOME/recycle/holdpredout 
+    /bin/rm -f $FVHOME/recycle/holdpredout/*lfo_Nx+-*
     /bin/mv -f $FVWORK/*lfo_Nx+-* $FVHOME/recycle/holdpredout/. 
-    /bin/cp $FVHOME/recycle/holdpredout/*lfo_Nx+-*  $LDHOME/input/met_forcing/. 
+    /bin/cp $FVHOME/recycle/holdpredout/*lfo_Nx+-*  $LDHOME/input/met_forcing/.
```

Comment:
Why should we merge "Nx+-" now, when we're already committed to switching to a "bkg" HISTORY Collection?